service/src/main/java/org/cbioportal/service/impl/MutationServiceImpl.java

```diff
@@ -42,7 +42,7 @@
         List<Mutation> mutationList = mutationRepository.getMutationsInMolecularProfileBySampleListId(molecularProfileId,
             sampleListId, entrezGeneIds, snpOnly, projection, pageSize, pageNumber, sortBy, direction);
```

Comment:
remove commented out lines if not using them anymore

Reply:
done.